psydac/polar/c1_projections.py

```diff
@@ -59,11 +59,13 @@ def compute_lambda( self, mapping ):
             y_ext = mapping.control_points[0:2,:,1]
 
             # Exclude ghost regions for calculations
-            x = x_ext[:, s2:e2+1]
-            y = y_ext[:, s2:e2+1]
+
+            x = x_ext[:, 0:e2-s2+1]
+            y = y_ext[:, 0:e2-s2+1]
```

Comment:
I get this error if I make it as before
```
        s1, s2 = mapping.space.vector_space.starts
        e1, e2 = mapping.space.vector_space.ends
        p1, p2 = mapping.space.vector_space.pads
    
        if s1 == 0:
    
            # Number of coefficients
            n0 = 3
    
            # Extract control points, including ghost regions
            x_ext = mapping.control_points[0:2,:,0]
            y_ext = mapping.control_points[0:2,:,1]
    
            # Exclude ghost regions for calculations
    
            x = x_ext[:, s2:e2+1]
            y = y_ext[:, s2:e2+1]
    
            SQRT3     = np.sqrt(3.0)
            ONE_THIRD = 1.0/3.0
>           (x0,y0)   = (x[0,0],y[0,0])
E           IndexError: index 0 is out of bounds for axis 1 with size 0
```

Reply:
We did not have this error before. Has the meaning of `mapping.control_points` changed?

Reply:
no it should be the same as before

Reply:
I found the problem, the issue is that ` mapping.control_points[0:2,:,0]` gives a numpy array because we take a slice from the StencilVector which is a numpy array

Reply:
Yes, I know that the slice should return an array. The question is whether this array includes the ghost regions... According to the comment line, it should. So the correct code should probably read
```python
x = x_ext[:, p2 : -p2]
y = y_ext[:, p2 : -p2]
```
Does that make sense to you?

Reply:
yes, it should work

Reply:
done in [0bce24f](https://github.com/pyccel/psydac/pull/213/commits/0bce24faec5301b27f539c21fc9144e83b1dffcf)